cymetric/metrics.py

```diff
@@ -230,6 +230,38 @@ def agents(series):
 del _agentsdeps, _agentsschema
 
 
+# Transaction Masses [kg]
+_transdeps = [
+    ('Materials', ('SimId', 'ResourceId', 'ObjId', 'TimeCreated'), 'Mass'), 
+    ('Transactions', ('SimId', 'TransactionId', 'SenderId', 'ReceiverId', 
+        'ResourceId'), 'Commodity')
+    ]
+
+_transschema = [
+    ('SimId', ts.UUID), ('TransactionId', ts.INT), 
+    ('ResourceId', ts.INT), ('ObjId', ts.INT), 
+    ('TimeCreated', ts.INT), ('SenderId', ts.INT), 
+    ('ReceiverId', ts.INT), ('Commodity', ts.STRING), 
+    ('Mass', ts.DOUBLE)
+    ]
+
+@metric(name='TransactionQuant', depends=_transdeps, schema=_transschema)
```

Comment:
Probably.  But that is a separate PR.  Unless an abbreviation is approved by the style guide or in the docs, it should not be used.  Currently we have no approved abbreviations.